### ISSUE TITLE
How to access accessibility_caption through the Python Module instaloader?

### ISSUE THREAD

**Issue by nOOBIE-nOOBIE** [instaloader/instaloader#355]
![image](https://user-images.githubusercontent.com/24412619/61452816-5fcfbf00-a97c-11e9-908c-9b922b480ced.png)
In the Json File i can see the accessibility_caption of the post. But cannot find anything on the documentation of instaloader as a module.

Is there something like `download_post` to download json of post like the cmd line returns so that i can parse myself. If someone can point something out, it would be of great help.

**Comment by nOOBIE-nOOBIE** [comment]
On saving the post meta data 
```L.save_metadata_json('testds',post)```
I can see the accessibility caption is present in the json file after saving.
![image](https://user-images.githubusercontent.com/24412619/61454173-d7ebb400-a97f-11e9-8d82-cc19391c963f.png)

How do i get this without saving and reading the JSON?

**Comment by nOOBIE-nOOBIE** [comment]
Okay i found a way. I was going through the code and couldnot understand much. But found that there is a function to save post,profile etc to json .
``` 
L.save_metadata_json('testds',post)
```
Going further i found another function ```._asdict()```.

It could be used with post object to get the dictionary of the post data. and it contained all information as in the original json

Here is a sample code

```
import instaloader
L = instaloader.Instaloader()
SHORTCODE = '7ENJQjDZT5'
post = Post.from_shortcode(L.context, SHORTCODE)
# getting post data from the instaloader library
post_url = 'https://www.instagram.com/p/' + post.shortcode
caption = post.caption
no_of_likes = post.likes
no_of_comments = post.comments
video_view = post.video_view_count
caption_hashtags = post.caption_hashtags
tags_in_captions = post.caption_mentions
url_photo_video = post.url

# getting accessibility_caption from the json
post_dict=post._asdict()
accessibility_caption = post_dict['accessibility_caption']

### do whatever you want with the data
```
